src/test/java/org/assertj/core/api/abstract_/AbstractAssert_satisfies_with_ThrowingConsumers_Test.java

```diff
@@ -17,32 +17,20 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.catchThrowable;
 import static org.assertj.core.api.BDDAssertions.then;
-import static org.assertj.core.api.BDDAssertions.thenNullPointerException;
+import static org.assertj.core.api.BDDAssertions.thenIllegalArgumentException;
 import static org.assertj.core.util.AssertionsUtil.expectAssertionError;
 import static org.assertj.core.util.ThrowingConsumerFactory.throwingConsumer;
 
 import java.nio.file.Path;
 import java.nio.file.Paths;
 
 import org.assertj.core.api.ThrowingConsumer;
-import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
-class AbstractAssert_satisfies_with_ThrowingConsumer_Test {
-
-  private ThrowingConsumer<Path> isEOF;
-
-  @BeforeEach
-  void setup() {
-    isEOF = path -> {
-      assertThat(isReadable(path)).isTrue();
-      // this would not compile if isEOF was declared as Consumer<Path> since it can throw an IOException
-      assertThat(readAllLines(path)).isEmpty();
-    };
-  }
+class AbstractAssert_satisfies_with_ThrowingConsumers_Test {
```

Comment:
use `hasMessageNotContaining​` and `hasMessageContaining​` whenever possible in these tests